src/tests/ftest/util/command_utils.py

```diff
@@ -266,7 +266,7 @@ def get_subprocess_state(self, message=None):
                 r"\d+\s+([DRSTtWXZ<NLsl+]+)\s+\d+", result.stdout)
         return state
 
-    def get_output(self, method_name, **kwargs):
+    def get_output(self, method_name, regex_method=None, **kwargs):
```

Comment:
It would be great if you can update the description. It looks like a big change.

Reply:
Will do.